acestep/acestep_v15_pipeline.py

```
@@ -321,87 +291,32 @@ def main():
-
-        allowed_paths = [output_dir]
-        for p in args.allowed_path:
-            if p and p not in allowed_paths:
-                allowed_paths.append(p)
-
-        # Enable API endpoints if requested
-        if args.enable_api:
-            print("Enabling API endpoints...")
-            from acestep.gradio_ui.api_routes import setup_api_routes
+        # Set environment variables for API server to pick up
+        if args.config_path: os.environ["ACESTEP_CONFIG_PATH"] = args.config_path
```

Ruff (0.14.14)

[error] Multiple statements on one line (colon)

(E701)